src/core/files.py

```diff
@@ -243,6 +243,33 @@ def handle_email_file(_file, kind, owner, label=None):
     return new_file
 
 
+def handle_multiple_email_files(request_files, file_owner):
+    """Processes in-memory files from a HTTP response and returns saved files.
+
+    Args:
+        request_files (list): in-memory files from a Django http response.
+        file_owner(django.contrib.auth.models.User): the owner of the saved files.
+
+    Returns:
+        list: file objects that can be sent as email attachemnts
+
+    """
+    attachments = []
+    if request_files:
```

Comment:
Is this necessary? If request files is a list, then you don't need to check if it contains anything before trying to loop through it

Reply:
Well spotted. I copied this over without really thinking.